template/package.json

```diff
@@ -33,6 +34,10 @@
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "react-intl": "^6.0.5",
+        "@svgr/cli": "^6.5.1",
```

Comment:
👀

Reply:
shouldn't be removed since `npx @svgr/cli` is used?

Reply:
yes but the previous changes wasnt pulled here automatically